src/main/java/frc/robot/commands/intakeCommands/IntakeMoveCommand.java

```diff
@@ -16,12 +17,18 @@ public class IntakeMoveCommand extends Command {
 
   double setPoint;
   boolean toggleEnable;
+  int slot;
+  double feedForward;
+  Timer timer = new Timer();
 
-  public IntakeMoveCommand(Intake intake, boolean toggleEnable, double setPoint) {
+  public IntakeMoveCommand(
```

Comment:
Can we switch the `toggleEnable`, `setPoint`, `slot`, and `feedForward` parameters to be part of an enum class and just pass an instance in here? It's really hard to tell what the intention is at the calling locations.